zadaci/sa-kolokvijuma/2020/E1/T34/G9/index.rst

```diff
@@ -0,0 +1,17 @@
+.. index::
+   Datoteke ; Tekstualne
+   Statičke strukture ; Niz
+
+
+Zadatak "Banka"
+=============================
```

Comment:
Please change this to a more descriptive name, for example "Obrada zahteva za kredit u banci". Be sure to have as = as your new title length is.